pkg/trivy/wrapper.go

```diff
@@ -82,6 +82,9 @@ func (w *wrapper) Run(imageRef string, auth RegistryAuth) (report ScanReport, er
 			fmt.Sprintf("TRIVY_USERNAME=%s", auth.Username),
 			fmt.Sprintf("TRIVY_PASSWORD=%s", auth.Password))
 	}
+	if insecureRegistry {
+		cmd.Env = append(cmd.Env, fmt.Sprintf("TRIVY_NON_SSL=true"))
```

Comment:
Note: This value might be changed after this PR.
https://github.com/aquasecurity/trivy/pull/398/files#diff-2e7777a3471e67a4473308ee6e4d19caR13